src/crimson/os/seastore/seastore.cc

```diff
@@ -1818,7 +1824,30 @@ SeaStore::Shard::_do_transaction_step(
       case Transaction::OP_TOUCH:
       {
         DEBUGT("op CREATE/TOUCH, oid={} ...", *ctx.transaction, oid);
```

Comment:
> Can OP_CREATE/TOUCH create a clone of an existing object, without OP_CLONE ?

When taking clone range, it will first touch the to be recovered onode.

Reply:
> > Can OP_CREATE/TOUCH create a clone of an existing object, without OP_CLONE ?
> 
> When taking clone range, it will first touch the to be recovered onode.

Actually, there's circumstances in which Recovery directly creates a clone of an existing object.

Reply:
Thanks! Then let's split this debug log into two different cases.